app/stac_api/models.py

```diff
@@ -624,17 +635,25 @@ def from_db(cls, db, field_names, values):
     def __str__(self):
         return self.name
 
+    def clean(self):
+        if self._original_values.get(
+            "name"
+        ) is not None and self.name != self._original_values.get("name"):
```

Comment:
I find surrounding `if .. and .. :` with brackets easier to read:
```
if (
    self._original_values("name") is not None and 
    self.name != self._original_values.get("name")
):
```

Reply:
changed